base_report_docx/report/report_docx.py

```diff
@@ -340,7 +340,7 @@ def _get_output_type(self, cr, uid, context):
         return action.output_type
 
     def _create_temp_folder(self, tmp_folder_name):
-        cmd = 'mkdir ' + tmp_folder_name
+        cmd = 'mkdir -p ' + tmp_folder_name
```

Comment:
Do you need to collect your garbage (even though this is tmp)?
cc @seb-elico

Reply:
The files created in /tmp will delete when pdf created. and pdf file will put in /printer